sailfishusermanager.h

```diff
@@ -49,6 +49,9 @@ public slots:
     void modifyUser(uint uid, const QString &new_name);
     void setCurrentUser(uint uid);
     uint currentUser();
+    QStringList usersGroups(uint uid);
+    void usersGroupsAdd(uint uid, const QString& group);
```

Comment:
Maybe just addGroup(uint uid, const QString &group), I'm assuming that this doesn't a thing i.e. succeeds if user alredy in the group.

For symmetry: removeGroup(uint uid, const QString& group)

Nit: ref should be on the group side :)

Reply:
Renamed